organization/global/iam/policies/route_53_access/terragrunt.hcl

```
@@ -0,0 +1,39 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "tfr:///terraform-aws-modules/iam/aws//modules/iam-policy?version=5.11.1"
+}
+
+dependency "xayn_domain" {
+  config_path = "../../../../eu-central-1/domains/xayn"
+}
+
+inputs = {
+  name        = "route_53_management"
+  path        = "/"
+  description = "Gives abiility to access route53 zones"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "route53:ChangeResourceRecordSets",
+        "route53:ListResourceRecordSets",
+        "route53:ListHostedZones",
+        "route53:GetHostedZone",
+        "route53:GetChange",
+        "route53:ListTagsForResource"
+      ],
+      "Effect": "Allow",
+      "Resource": [
+        "*"
+      ]
+    }
+  ]
+}
+EOF
```

when we use `jsonencode` we get formatting for free
```suggestion
  policy = jsonencode(
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Action": [
        "route53:ChangeResourceRecordSets",
        "route53:ListResourceRecordSets",
        "route53:ListHostedZones",
        "route53:GetHostedZone",
        "route53:GetChange",
        "route53:ListTagsForResource"
      ],
      "Effect": "Allow",
      "Resource": [
        "*"
      ]
    }
  ]
})
```